### ISSUE TITLE
ManifestSummary/Tooltip/Popover accessibility 

### ISSUE THREAD

**Issue by majornista** [contentauth/c2pa-js-legacy#145]
Tooltip component used in ManifestSummary should:

1. Trigger should be keyboard accessible as a button.
2. Trigger should be labeled by the Tooltip and the header.
3. Popover for Tooltip should expand on focus and close on blur.
4. Popover should hide with Escape key and toggle on Trigger click.
5. Fix text color contrast for --cai-secondary-color.
6. Fix color contrast for  --cai-icon-fill
7. Add focus outline color and outline-offset to Tooltip button and View More
8. Use `<section>` with aria-labelledby for PanelSection.
9. Add role="heading" and a heading level for the .heading-text element within PanelSection
10. Add role="img" and appropriate aria-label for each of the Icons
